src/tsickle.ts

```diff
@@ -654,6 +654,9 @@ class Annotator extends ClosureRewriter {
 
   private visitClassDeclaration(classDecl: ts.ClassDeclaration) {
     let jsDoc = this.getJSDoc(classDecl) || [];
+    // Always include @unrestricted so that Closure compiler does not complain if some class fields
+    // are not assigned in the constructor.
```

Comment:
Add that every tsickle output class does that because of '_tsickle_typeAnnotationsHelper'.